### ISSUE TITLE
Cannot SELECT table.* ...

### ISSUE THREAD

**Issue by danClauz** [cube2222/octosql#158]
I try to execute query like

`SELECT dyusr.* FROM dynamo.departments as dydep
LEFT JOIN dynamo.user_role as dyur ON dydep.id = dyur.department_id
LEFT JOIN dynamo.users as dyusr ON dyusr.id = dyur.user_id
WHERE dydep.id = 1`

with expectation it will returns only users data, but it return all of tables field value

**Comment by danClauz** [comment]
`SELECT dyusr.* FROM dynamo_departments as dydep LEFT JOIN dynamo_user_role as dyur ON dydep.id = dyur.department_id LEFT JOIN dynamo_users as dyusr ON dyusr.id = dyur.user_id WHERE dydep.id = 1`

**Comment by cube2222** [comment]
Indeed, such expressions do not work at this time, and OctoSQL seems to interpret this as a simple SELECT * FROM ... query. I've changed the title to be a feature request. This shouldn't be hard to add, and will surely make it into the next release.
